### PR TITLE
removed `>>>` and `...` in favour of whitespace

### PR DESCRIPTION
**Pull request by venthur**
In some cases there were return values like so

    >>> foo
    200

I've replaced them with

    foo
    # 200

and so on

partially fixes: #2223

### CODE REVIEW

File: docs/advanced-usage.rst [hunk 1]
```diff
@@ -80,33 +80,33 @@ chunks of the response content.
...
-    ...     'GET',
-    ...     'http://httpbin.org/bytes/1024',
-    ...     preload_content=False)
-    >>> r.read(4)
-    b'\x88\x1f\x8b\xe5'
+    r = http.request(
+        'GET',
+        'http://httpbin.org/bytes/1024',
+        preload_content=False)
+    r.read(4)
```
Comment (ubdussamad) [inline docs/advanced-usage.rst:87]: `print(r.read())`

File: docs/advanced-usage.rst [hunk 2]
```diff
@@ -80,33 +80,33 @@ chunks of the response content.
...
-    >>> json.load(reader(r))
-    {'origin': '127.0.0.1'}
-    >>> r.release_conn()
+    import codecs
+    reader = codecs.getreader('utf-8')
+    r = http.request(
+        'GET',
+        'http://httpbin.org/ip',
+        preload_content=False)
+    json.load(reader(r))
```
Comment (ubdussamad) [inline docs/advanced-usage.rst:107]: print ....

File: docs/index.rst [hunk 3]
```diff
@@ -33,13 +33,13 @@ urllib3 is powerful and easy to use:
...
-    >>> r = http.request('GET', 'http://httpbin.org/robots.txt')
-    >>> r.status
-    200
-    >>> r.data
-    'User-agent: *\nDisallow: /deny\n'
+    import urllib3
+    http = urllib3.PoolManager()
+    r = http.request('GET', 'http://httpbin.org/robots.txt')
+    r.status
+    # 200
```
Comment (ubdussamad) [inline docs/index.rst:40]: +

**Review by ubdussamad — COMMENTED** [review]
Please add `print` to all files.

**Comment by sethmlarson** [comment]
@venthur As a part of this issue it'd be good to do the splitting of imports and code into sections since we're touching almost every example anyways? What are your thoughts on this?

**Comment by venthur** [comment]
@SethMichaelLarson Sure, just give me one example so I don't mess it up and I'll do it tomorrow.

**Comment by venthur** [comment]
I've separated the imports from the rest of the code now.

**Comment by sethmlarson** [comment]
@venthur After merging https://github.com/urllib3/urllib3/pull/2231 I think most of the changes in this PR aren't needed any more. Should we close this one and fix other areas of the documentation in a new PR?

**Pull request closed without merging**